bin/run

```diff
@@ -15,17 +15,22 @@ usage() {
 case $1 in
   localweb)
     python manage.py migrate
-    python manage.py runserver 0.0.0.0:${PORT}
+    exec python manage.py runserver 0.0.0.0:${PORT}
```

Comment:
Without it the process doesn't become process ID 1 in the container which would prevent it from correctly receiving posix signals which is required to properly shutdown when the container is stopped or killed.